docs/reference/api.md

```diff
@@ -9,6 +9,28 @@ The API can be accessed by [various clients] from programming languages or the c
 [Buf Registry]: https://buf.build
 [various clients]: clients.md
 
+## Authentication
+
+In order to connect to the API, you must provide a [Bearer Token] in order to identify yourself.
+If you are using an officially supported client library, refer to its documentation or examples for how to provide your credential.
+If you are using your own or third party gRPC clients, refer to the [gRPC Authentication documentation].
+
+Authzed users can find their API Token in the [Authzed dashboard].
+
+[Bearer Token]: https://datatracker.ietf.org/doc/html/rfc6750#section-2.1
+[gRPC Authentication documentation]: https://grpc.io/docs/guides/auth
+[Authzed dashboard]: https://app.authzed.com
+
+## Alternative HTTP/JSON API
+
+By configuring an additional port to listen for traffic, SpiceDB can also support handling JSON requests over HTTP.
```

Comment:
The "additional port" part is a bit vague. Could mention the specific `http` flag or just to look at spicedb's available flags. Something actionable for someone who wants to run this.

Reply:
This isn't a doc for configuring SpiceDB so I do not want to cover CLI flags here.

Reply:
In the future, we can link to that doc to make it actionable.